rust/src/lib.rs

```diff
@@ -720,7 +714,7 @@ pub extern "C" fn zcashlc_get_verified_balance(
         let db_data = wallet_db(db_data, db_data_len, network)?;
         if account >= 0 {
             (&db_data)
-                .get_target_and_anchor_heights()
+                .get_target_and_anchor_heights(ANCHOR_OFFSET)
```

Comment:
ANCHOR_OFFSET is defined as 10. Is this the same as the previous behaviour?

Reply:
Ah you said below that it is, @nuttycom.

Reply:
Yes. It was hardcoded inside librustzcash.